test/framework/deployments/tracing/deployment.go

```diff
@@ -0,0 +1,47 @@
+package tracing
+
+import (
+	"github.com/pkg/errors"
+
+	"github.com/kumahq/kuma/test/framework"
+)
+
+type Tracing interface {
+	ZipkinCollectorURL() string
+	TracedServices() ([]string, error)
+}
+
+type Deployment interface {
+	framework.Deployment
+	Tracing
+}
+
+const DeploymentName = "tracing"
+
+func From(cluster framework.Cluster) Tracing {
+	return cluster.Deployment(DeploymentName).(Tracing)
+}
+
+func Install() framework.InstallFunc {
+	return func(cluster framework.Cluster) error {
+		switch c := cluster.(type) {
+		case *framework.K8sCluster:
+			deployment := &k8SDeployment{}
+			c.Deployments[DeploymentName] = deployment
```

Comment:
I don't really like these direct writes to the map. Can we extend `framework.Deployment` interface with a new method `Name() string` and add a new method to `framework.Cluster` interfaces:
```go
func (c *K8sCluster) Deploy(d Deployment) error {
    c.deployment[d.Name()] = d
    return d.Deploy(c)
}
```